df

```diff
@@ -0,0 +1,144 @@
+[33mcommit caea003545f73969d95c6136c37eb4d5d6157055[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;31morigin/master[m[33m, [m[1;31morigin/HEAD[m[33m)[m
```

Comment:
Did you mean to commit this file? I don't think we want it in the repo.

Reply:
I've no idea what that even is; Git created it and pulled it in automatically, dunno what it does or what could break without it.

Reply:
It doesn't appear to be any kind of git metadata file as far as I can tell.. perhaps just a temporary thing. I think it's safe to remove.